wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss

```diff
@@ -243,24 +240,33 @@
     &-heading {
       display: flex;
       gap: rem(12);
-      justify-content: center;
+      justify-content: start;
       align-items: center;
       margin-bottom: rem(24);
-      @extend %text-style-heading-large-strong-caps;
+      @extend %text-style-heading-large-strong;
+      color: $color-accent-primary-blue-100;
+    }
 
-      @include breakpoint('small') {
-        margin-bottom: rem(32);
-      }
+    &-related-posts,
+    &-related-terms {
+      @include breakpoint('medium') {
+        display: inline-block;
 
-      @include breakpoint('large') {
-        margin-bottom: rem(40);
-      }
+        &:nth-of-type(1) {
+          width: calc(100% * 2 / 3);
+        }
 
-      .icon {
-        height: 0.5em;
-        font-size: 2em;
+        &:nth-of-type(2) {
+          width: calc(100% / 3);
+        }
```

Comment:
The transition from one column to two happens at the `large` breakpoint.  Also, the split is `auto` and `260px`.  I recommend using grid on the section and setting `grid-template-columns` to `auto rem(260)`